<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDialog</class>
 <widget class="QDialog" name="ViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_host">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HOSTELS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_uni">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UNIVERSITIES</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_teach">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TEACHERS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PLZ SELECT ONE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VIEW</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
